ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/CheckpointRange.java

```diff
@@ -59,4 +59,8 @@ public String toString() {
         .add("end", end.getNumber())
         .toString();
   }
+
+  public int getSegmentLength() {
+    return (int) (end.getNumber() - start.getNumber());
```

Comment:
This case is safe because the gap between checkpoint headers is originally specified as an int but having the explicit check probably makes sense anyway.  So I've changed it to use toIntExact.